packages/gatsby-plugin-preact/src/gatsby-node.js

```diff
@@ -24,6 +24,25 @@ exports.onCreateWebpackConfig = ({ stage, actions, getConfig }) => {
     })
   }
 
+  // add preact to the framework bundle
+  if (stage === `build-javascript`) {
+    const webpackConfig = getConfig()
+    if (
+      webpackConfig?.optimization?.splitChunks?.cacheGroups?.framework?.test
+    ) {
+      const frameworkRegex =
+        webpackConfig?.optimization?.splitChunks?.cacheGroups?.framework?.test
```

Comment:
```suggestion
        webpackConfig.optimization.splitChunks.cacheGroups.framework.test
```

you check for existence in `if` condition - so no need to use optional chaining here again? This is especially "visible" when in next statement we assign something else here